DataHubService.js

```diff
@@ -0,0 +1,114 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import axios from 'axios';
+
+const headers = {Accept: 'application/ld+json, application/json'};
+
+export class DataHubService {
+  constructor({
+    urls = {
+      base: '/data-hubs'
+    }
+  } = {}) {
+    this.config = {urls};
+  }
+
+  /**
+   * @method create
+   * @param {Object} options
+   * @param {string} options.baseUrl - the base baseUrl
```

Comment:
baseUrl optional and called url.

Reply:
It's baseUrl elsewhere, maybe the code in this function should be renamed vs param.

Reply:
Fixed...enough